app/views/home/_about.md

```diff
@@ -21,7 +21,7 @@ A huge thank you to [Google Australia](http://www.google.com.au) whose donation
 
 ### <a name='contribute'></a>You can help
 
-Make this project better for everyone by [summarising divisions](/divisions) and [maintaining and discussing policies](/policies). You can also [make a tax deductible donation](https://www.openaustraliafoundation.org.au/donate/theyvoteforyou/) to the OpenAustralia&nbsp;Foundation to support us in creating and maintaining projects like this.
+Make this project better for everyone by [summarising divisions](/help/research/) and [maintaining and discussing policies](/policies). You can also [make a tax deductible donation](https://www.openaustraliafoundation.org.au/donate/theyvoteforyou/) to the OpenAustralia&nbsp;Foundation to support us in creating and maintaining projects like this.
```

Comment:
This is a good change but it's not related to the other changes in this PR, you should remove this commit from the branch. I'm happy to explain how to do this if you like :)